toplevel/genprintval.ml

```diff
@@ -589,6 +589,8 @@ module Make(O : OBJ)(EVP : EVALPATH with type valu = O.t) = struct
               tree_of_val (depth - 1) obj ty
           | Tpackage _ ->
               Oval_stuff "<module>"
+          | Tof_kind _jkind ->
+              Oval_stuff "of_kind_ <kind>"
```

Comment:
This really should never happen, I think -- it would be a value of type `Tof_kind blah`. I think this could `fatal_error`.